app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+
+def cache(func: callable) -> any:
+    results = {}
+
+    def inner(*args) -> any:
+        if args in results.keys():
```

Comment:
`in` keyword as well as `for` loop, for example, are iterating dictionary keys by default. So you don't need to write .keys()